### ISSUE TITLE
Update demo to reflect cmor 3.2.8 changes for history and variable attributes

### ISSUE THREAD

**Issue by durack1** [PCMDI/obs4MIPs-cmor-tables#128]
With CMOR 3.2.8 a number of changes have been made to deal with standing issues, such as better control of the history attribute (PCMDI/cmor#238), and extending the cmor python bindings to allow for variable attributes of non-string types to be written (PCMDI/cmor#244).

There is a need to augment the demo to provide user examples of this new functionality.

**Comment by durack1** [comment]
Relevant examples are linked from https://github.com/PCMDI/cmor/issues/238#issuecomment-339787693

**Comment by durack1** [comment]
These issues are now resolved in the latest nightly (`2017.11.01.3.2.8-np19py27h1e94580_0`), with the current output:
```
netcdf prw_mon_REMSS-PRW-6-6-0_BE_gn_198701-198812 {
dimensions:
	time = UNLIMITED ; // (24 currently)
	lat = 72 ;
	lon = 144 ;
	bnds = 2 ;
variables:
	double time(time) ;
		time:bounds = "time_bnds" ;
		time:units = "days since 1987-1-1 0:0:0" ;
		time:calendar = "gregorian" ;
		time:axis = "T" ;
		time:long_name = "time" ;
		time:standard_name = "time" ;
	double time_bnds(time, bnds) ;
	double lat(lat) ;
		lat:bounds = "lat_bnds" ;
		lat:units = "degrees_north" ;
		lat:axis = "Y" ;
		lat:long_name = "latitude" ;
		lat:standard_name = "latitude" ;
	double lat_bnds(lat, bnds) ;
	double lon(lon) ;
		lon:bounds = "lon_bnds" ;
		lon:units = "degrees_east" ;
		lon:axis = "X" ;
		lon:long_name = "longitude" ;
		lon:standard_name = "longitude" ;
	double lon_bnds(lon, bnds) ;
	float prw(time, lat, lon) ;
		prw:standard_name = "atmosphere_water_vapor_content" ;
		prw:long_name = "Water Vapor Path" ;
		prw:comment = "vertically integrated through the atmospheric column" ;
		prw:units = "kg m-2" ;
		prw:cell_methods = "area: time: mean" ;
		prw:missing_value = 1.e+20f ;
		prw:_FillValue = 1.e+20f ;
		prw:valid_min = 2.f ;
		prw:valid_max = 3.f ;

// global attributes:
		:Conventions = "CF-1.7 ODS-2.1" ;
		:activity_id = "obs4MIPs" ;
		:contact = "RSS (support@remss.com)" ;
		:creation_date = "2017-11-08T20:36:13Z" ;
		:data_specs_version = "2.1.0" ;
		:frequency = "mon" ;
		:further_info_url = "https://furtherinfo.es-doc.org/obs4MIPs.RSS.REMSS-PRW.REMSS-PRW-6-6-0.prw" ;
		:grid = "2.5x2.5 degree latitude x longitude" ;
		:grid_label = "gn" ;
		:history = "2017-11-08T20:36:13Z; CMOR rewrote data to be consistent with obs4MIPs, and CF-1.7 ODS-2.1 standards" ;
		:institution = "Remote Sensing Systems, Santa Rosa, CA 95401, USA" ;
		:institution_id = "RSS" ;
		:nominal_resolution = "250 km" ;
		:product = "observations" ;
		:realm = "atmos" ;
		:region = "global" ;
		:source = "REMSS-PRW 6.6.0 (2017): Water Vapor Path" ;
		:source_id = "REMSS-PRW-6-6-0" ;
		:source_label = "REMSS-PRW" ;
		:source_type = "satellite_blended" ;
		:source_version_number = "6.6.0" ;
		:table_id = "obs4MIPs_Amon" ;
		:table_info = "Creation Date:(04 October 2017) MD5:9516b952d4e1c5084ec8c8e481e6cf35" ;
		:title = "PRW v6.6.0 prepared for obs4MIPs (ODS-v2.1.0)" ;
		:tracking_id = "hdl:21.14102/0dc9218d-6923-4415-8fb0-93cd9fa734c1" ;
		:variable_id = "prw" ;
		:variant_info = "Best Estimate" ;
		:variant_label = "BE" ;
		:license = "Data in this file produced by <Your Centre Name> is licensed under a Creative Commons Attribution-ShareAlike 4.0 International License (https://creativecommons.org/licenses/). Use of the data must be acknowledged following guidelines found at <a URL maintained by you>. Further information about this data, including some limitations, can be found via <some URL maintained by you>" ;
		:cmor_version = "3.2.8" ;
}
```

**Comment by durack1** [comment]
Fixed by #130
